functions.md

```diff
@@ -120,12 +134,39 @@ const squareNum = (num) => {
 
 We can refactor the function to:
 
-```js    
-const squareNum = num => num * num;
+```js
+const squareNum = (num) => num * num;
 ```
 
 Notice the following changes:
 
 - The parentheses around num have been removed, since it has a single parameter.
 - The curly braces { } have been removed since the function consists of a single-line block.
-- The return keyword has been removed since the function consists of a single-line block.
\ No newline at end of file
+- The return keyword has been removed since the function consists of a single-line block.
+
+A typical example of function and it's implementation.
+
+```js
+let factorial = (n) => {
```

Comment:
Good job adding a function declaration, it will be better to include the function call too including the results in comment, this illustrates the full functionality of a function.

Reply:
done

Reply:
Done? 👁